### ISSUE TITLE
Python versions with `rc...` suffix are not recognized

### ISSUE THREAD

**Issue by ulf1** [huggingface/huggingface_hub#79]
Error Message

```
../lib/python3.8/site-packages/huggingface_hub/file_download.py", line 37, in <genexpr>
    if tuple(int(i) for i in _PY_VERSION.split(".")) < (3, 8, 0):
ValueError: invalid literal for int() with base 10: '3rc1'
```

My Python version: `3.8.3rc1`

**Comment by julien-c** [comment]
I think this might be fixed by a variation of #27 

Anyone wants to tackle this?

**Comment by ulf1** [comment]
> I think this might be fixed by a variation of #27
> 
> Anyone wants to tackle this?

https://github.com/huggingface/huggingface_hub/pull/82
